src/pseudo_intrinsics.h

```diff
@@ -45,6 +45,7 @@ typedef __m512i vtype;
 #define vand                    _mm512_and_si512
 #define vandnot                 _mm512_andnot_si512
 #define vcmov(y, z, x)          vxor(z, vand(x, vxor(y, z)))
+#define vcmpeq_epi8_mask        (uint64_t)_mm512_cmpeq_epi8_mask
```

Comment:
I found the following typedefs in icc's headers:
```
typedef unsigned char       __mmask8;
typedef unsigned short      __mmask16;
typedef unsigned int        __mmask32;
typedef unsigned __int64    __mmask64;
```
So there's actually no need for something like `vmask2int`.